local-cli/generator-macos/templates/react-native.config.js

```diff
@@ -1,33 +1,9 @@
-/**
- * This cli config is needed for the coexistance of react-native and other
- * out-of-tree implementations such react-native-macos.
- * The following issue is tracked by
- * https://github.com/react-native-community/discussions-and-proposals/issues/182
- *
- * The work-around involves having a metro.config.js for each out-of-tree
- * platform, i.e. metro.config.js for react-native and
- * metro.config.macos.js for react-native-macos.
- * This react-native.config.js looks for a --use-react-native-macos
- * switch and when present pushes --config=metro.config.macos.js
- * and specifies reactNativePath: 'node_modules/react-native-macos'.
- * The metro.config.js has to blacklist 'node_modules/react-native-macos',
- * and conversely metro.config.macos.js has to blacklist 'node_modules/react-native'.
- */
-'use strict';
-
 const macSwitch = '--use-react-native-macos';
-const windowsSwitch = '--use-react-native-windows';
 
 if (process.argv.includes(macSwitch)) {
   process.argv = process.argv.filter(arg => arg !== macSwitch);
   process.argv.push('--config=metro.config.macos.js');
   module.exports = {
     reactNativePath: 'node_modules/react-native-macos',
   };
-} else if (process.argv.includes(windowsSwitch)) {
-  process.argv = process.argv.filter(arg => arg !== windowsSwitch);
-  process.argv.push('--config=metro.config.windows.js');
-  module.exports = {
-    reactNativePath: 'node_modules/react-native-windows',
-  };
 }
```

Comment:
Never used `fs.appendFile()`. I assume it will add a new line here before writing the extra data?

Reply:
Seems it does 👍 

```
$ echo 'foo' > test.txt
$ node
Welcome to Node.js v12.7.0.
Type ".help" for more information.
> const fs = require('fs')
undefined
> fs.appendFileSync('test.txt', 'bar')
undefined
> 
(To exit, press ^C again or ^D or type .exit)
> 
$ cat test.txt 
foo
bar
```